_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,226 @@
+
+However, fine-tuning tasks often require extensive manual intervention, including the configuration of training environments and the distribution of data across nodes. The new [Fine-Tune API](https://www.kubeflow.org/docs/components/training/user-guides/fine-tuning/) aims to simplify this process, offering an easy-to-use Python interface that abstracts away the complexity involved in setting up and executing fine-tuning tasks on distributed systems.
+
+By providing this API, Training Operator not only simplifies the user experience for ML practitioners but also leverages its existing infrastructure for distributed training. You can take advantage of Kubernetes’ ability to dynamically schedule GPU, thus saving on compute resources and cost. Training Operator also gives you fault tolerant guarantees, to save your training procedures from cluster node failures.
+
+## Pipelines
+
+### Argo Workflows and Tekton Backends Consolidation
+
+The Pipelines Tekton backend has been [merged](https://github.com/kubeflow/pipelines/pull/10678) into the main Kubeflow Pipelines repository. You can now choose what workflow engine to use from the same Pipelines version. This proves the extensibility and flexibility of the KFP v2 architecture, which encourages other contributors to bring support for other workflow engines.
+
+Compared to Argo Workflows, Tekton provides better extensibility thanks to pipeline definitions and reusable components that facilitate easier customization and integration into various CI/CD systems, making it adaptable for diverse machine learning workflows. Tekton also optimizes resource usage and execution efficiency, enhancing scalability for large-scale machine learning workloads compared to Argo Workflows, resulting in faster execution and better resource management.
+
+In this [blog post](https://developer.ibm.com/blogs/awb-tekton-optimizations-for-kubeflow-pipelines-2-0/) you can find more details about the benefits of running KFP with Tekton instead of Argo Workflows. 
+
+### Argo Workflows Upgrade
+
+Kubeflow Pipelines’s Argo Workflows backend is [upgraded to 3.4.16](https://github.com/kubeflow/pipelines/issues/10469). This upgrade moves the supported version closer to the latest upstream version and brings lots of CVE resolutions. The previous version was no longer being patched by the Argo community, so lots of security issues had accumulated over time.
+
+### v1 Feature Parity
+
+We made a huge progress towards KFPv1 feature parity by adding more Kubernetes  resources to the Pipelines code with the new kfp-kubernetes 1.2.0 Python package.
+
+## Katib
+
+Kubeflow 1.9 ships with Katib 0.17, which brings [official support](https://github.com/kubeflow/katib/pull/2315) for ARM64, getting us one step closer to full ARM64 coverage.
```

Comment:
should we have a short intro into Katib in the 1st sentence?   i.e. ships with Katib 0.17, and extends Katib's fine tuning capabilities for LLMs and ML models.   2nd comment .   Is ARM64 the most important feature ?  should it come later  ?   Could ARM update start on line 64.

Reply:
LLM fine tuning is under training operator. I will let @andreyvelich comment on what the most prominent feature for this release should be. ARM64 support is a long time coming feature. It appears to me that other SDK improvements do not introduce radical new features, rather they are minor improvements.

Reply:
Yes, Hyperparameter optimization during LLM fine-tuning will be added only in the next Katib release.
@helenxie-bit and @kubeflow/wg-training-leads are working on that as part of this: https://github.com/kubeflow/katib/issues/2339.